### PR TITLE
gui: versioner->versioning and add missing translations

### PR DESCRIPTION
**Pull request by imsodin**
All over the UI and docs we use "versioning", except for this one instance where it is "versioner". Minor issue, but it caused [confusion for translation](https://www.transifex.com/syncthing/syncthing/translate/#de/gui/117529706?q=external). Also strings for the new usage reporting were missing.

### CODE REVIEW

File: gui/default/syncthing/settings/settingsModalView.html [hunk 1]
```diff
@@ -142,7 +142,8 @@
             <div ng-if="tmpOptions.upgrades != 'candidate'">
               <label translate for="urVersion">Anonymous Usage Reporting</label> (<a href="" translate data-toggle="modal" data-target="#urPreview">Preview</a>)
               <select class="form-control" id="urVersion" ng-model="tmpOptions._urAcceptedStr">
-                <option ng-repeat="n in urVersions()" value="{{n}}">Version {{n}}</option>
+                  <option ng-repeat="n in urVersions()" value="{{n}}">{{'Version' | translate}} {{n}}</option>
+                  <!-- <option ng-repeat="n in urVersions()" value="{{n}}"><translate>Version</translate> {{n}}</option> -->
                 <!-- 1 does not exist, as we did not support incremental formats back then. -->
```
Comment (AudriusButkevicius) [inline gui/default/syncthing/settings/settingsModalView.html]: Whats up with the comments?

Comment (AudriusButkevicius) [reply gui/default/syncthing/settings/settingsModalView.html]: And indent?

**Comment by AudriusButkevicius** [comment]
This will cause everyone to have to retranslate, is it really worth it?

File: gui/default/syncthing/settings/settingsModalView.html (hunk 1 above)
Comment (imsodin) [reply gui/default/syncthing/settings/settingsModalView.html]: No idea what it is with the comment, not of my doing, I didn't even notice before. Indenting is my wrongly adjusted editor -> will fix.

**Comment by imsodin** [comment]
I do believe so. The two translations will be side by side, so it is a simple copy and paste and it will be both more consistent and less confusing. Often in translation meaning gets slightly altered even for consistent "initial state", so such little problems can already cause much bigger changes in meaning - that's what happened to the German translation which was simply wrong.

**Comment by calmh** [comment]
@st-review merge

gui: "versioner" -> "versioning" and add missing translations

**Comment by st-review** [comment]
:ok_hand: Merged as c704ba9ef9d18730d3c3a48f6c1ad78d9b16b950. Thanks, @imsodin!

**Pull request closed without merging**